plugins/legacy-preset-chart-nvd3/test/utils/tokenize.test.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { tokenizeToNumericArray, tokenizeToStringArray } from '../../src/utils/tokenize';
+
+describe('tokenizeToNumericArray', () => {
+  it('evals numeric strings properly', () => {
+    expect(tokenizeToNumericArray('1')).toStrictEqual([1]);
+    expect(tokenizeToNumericArray('1,2,3,4')).toStrictEqual([1, 2, 3, 4]);
+    expect(tokenizeToNumericArray('   1, 2,   3,    4 ')).toStrictEqual([1, 2, 3, 4]);
```

Comment:
Could throw some decimal values in here, if you wanna make sure people don't randomly turn parseFloat int parseInt or something silly :)

Reply:
good point 👍